src/main/java/com/sourcegraph/webhook/README.md

```diff
@@ -0,0 +1,66 @@
+# Webhooks
+The Sourcegraph Bitbucket plugin provides an internal implementation for webhooks that supports listening to events at a global or project scope.
+
+## Contents
+- [Payload](#payload)
+- [REST API](#rest-api)
+    * [List](#list)
+    * [Create](#create)
+    * [Delete](#delete)
+
+## Payload
+The event payload request is sent when an event is fired. There will be five attempts before the request fails. This will be recorded in the logs.
```

Comment:
>This will be recorded in the logs.

Does "This" refers to errors/failures only or every request?

Reply:
Successful requests are outputted as debug messages while failures will show up as an error. I added some clarification to the README.